controller/persistence/cli.py

```diff
@@ -0,0 +1,28 @@
+
+from mongo_client import Database
```

Comment:
This seems like a test script?
Not really suited for the controller code.
Maybe include in the utils folder located in the root folder of the repo